aptos-move/aptos-debugger/src/aptos_debugger.rs

```diff
@@ -5,21 +5,22 @@ use anyhow::{bail, format_err, Result};
 use aptos_gas_profiling::{GasProfiler, TransactionGasLog};
 use aptos_rest_client::Client;
 use aptos_types::{
-    account_address::AccountAddress,
-    state_store::TStateView,
-    transaction::{
+    account_address::AccountAddress, block_executor::config::{BlockExecutorConfig, BlockExecutorConfigFromOnchain, BlockExecutorLocalConfig}, state_store::TStateView, transaction::{
         signature_verified_transaction::SignatureVerifiedTransaction, SignedTransaction,
         Transaction, TransactionInfo, TransactionOutput, TransactionPayload, Version,
-    },
-    vm_status::VMStatus,
+    }, vm_status::VMStatus
 };
 use aptos_validator_interface::{
     AptosValidatorInterface, DBDebuggerInterface, DebuggerStateView, RestDebuggerInterface,
 };
-use aptos_vm::{data_cache::AsMoveResolver, AptosVM, VMExecutor};
+use aptos_vm::{aptos_vm::RAYON_EXEC_POOL, block_executor::{AptosTransactionOutput, BlockAptosVM}, data_cache::AsMoveResolver, AptosVM};
```

Comment:
We discussed before when RAYON_EXEC_POOL was moved out of block_executor to aptos_vm that we should have moved it back to aptos-vm/block_executor wrapper so the threadpool doesn't have other use-cases and then interfere with block execution. Debugger might be okay, but let's think about it a bit. For context:
https://github.com/aptos-labs/aptos-core/blame/e0662930743c280a4b7c51fad94256439a8ac1f7/aptos-move/aptos-vm/src/aptos_vm.rs#L119C13-L119C13